lib/src/code_field/code_controller.dart

```diff
@@ -97,7 +94,7 @@ class CodeController extends TextEditingController {
 
   CodeController({
     String? text,
-    Mode? language,
+    required Mode? language,
```

Comment:
`language` should not be required. It breaks backwards compatibility. If we want others to migrate to our editor, we must support as much use cases as possible.